### PR TITLE
Internationalization of ZIM viewer

### PR DESCRIPTION
**Pull request by veloman-yunkan**
This PR has been extracted out of #846 so that changes related to ZIM viewer can be merged without waiting for resolution of the issues discovered while testing internationalization of the welcome page.

Depends on #869

In this PR I have eliminated the roundabout way of arriving at the final state of the internationalization utilities. Despite the title of this PR, it is mostly about introducing the infrastructure for frontend internationalization - ZIM viewer serves only as a small test range.

### CODE REVIEW

**Comment by veloman-yunkan** [comment]
Cleaned-up portion of #846 related to reusable frontend internationalization utilities and their application in the context of ZIM viewer is ready for review and final testing.

File: static/skin/taskbar.css [hunk 1]
```diff
@@ -44,6 +44,10 @@
     margin: 0 auto;
 }
 
+#ui_language {
```
Comment (mgautierfr) [inline static/skin/taskbar.css:47]: Also a comment for the commit message:

Typos in the title : `lanuguage`

**Review by mgautierfr — CHANGES_REQUESTED** [review]
Two small changes for the commit messages but else we are good. It works pretty well.

File: static/resources_list.txt [hunk 2]
```diff
@@ -15,6 +15,7 @@ skin/fonts/Roboto.ttf
 skin/search_results.css
 skin/blank.html
 skin/viewer.js
+skin/mustache.min.js
```
[the remark this answers is not in the document]
Comment (veloman-yunkan) [reply static/resources_list.txt:20]: Done

File: static/skin/taskbar.css (hunk 1 above)
Comment (veloman-yunkan) [reply static/skin/taskbar.css:47]: Done

**Review by kelson42 — APPROVED** [review]
LGTM

**Review by mgautierfr — APPROVED** [review]

**Comment by veloman-yunkan** [comment]
This PR introduced another problem for the seamonkey browser caused by usage of [`import.meta`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta) in a620c8658b450f76aa8edbaf75016462c4eb9893

**Pull request merged**